portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -144,6 +126,31 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     response.getWriter().println(gson.toJson(LoadAllBlogsOrLast(/*all=*/false)));   
   }
  
+  private List<BlogMessage> putRepliesWithPosts(List<BlogMessage> blogMessages) {
+    
+ 
+    // TODO: Use helper functions for lines 73-101.
```

Comment:
I would generally avoid using very specific line numbers in TODOs because we will easily forget to update the line numbers when we change the code elsewhere, making us lose track of the numbers maybe in a day or two. Maybe try to use some slightly less specific description to refer to the range of code you mean.